scripts/data_collection/update_google_sheet.py

```diff
@@ -0,0 +1,113 @@
+"""Script to update Google sheet containing with user feedback and conversation history.
+
+In order to use this script, file with client secret to Google API must be present (symlinked etc.)
```

Comment:
Would be useful to say what exactly is needed.

Reply:
fixed @onmete